app/models/enqueuer.rb

```diff
@@ -0,0 +1,49 @@
+class Enqueuer
+  DELAY_THRESHOLD = 30
+  def enqueue_faxes
+    SnapApplication.
+      faxable.
+      untouched_since(DELAY_THRESHOLD.minutes.ago).
+      find_each do |snap_application|
+        create_and_enqueue_export(destination: :fax,
+                                  snap_application: snap_application)
+      end
+  end
+
+  def create_and_enqueue_export(params)
+    enqueue(Export.create(params))
+  end
+
+  def self.create_and_enqueue_export(params)
```

Comment:
can we put class methods above instance methods here? That's my general rule of thumb for class organization: class methods, instance methods (also dependency order, where methods that call other methods are above all methods they depend on)